### ISSUE TITLE
Feature Request: Have a custom home page component

### ISSUE THREAD

**Issue by boc-the-git** [madeofpendletonwool/PinePods#239]
Because I don't have an interest in "what are the most recent podcasts", the current home page is useless to me. I've then got several button presses before I could play a podcast I'm interested in.

A couple ideas:
- At the very least add a filter setting, that is also sortable
- Just replace the current home screen with the /podcasts screen; that's the UX I'm used to and find much preferable, though I can acknowledge there might be better options

**Comment by madeofpendletonwool** [comment]
Good call. I think it would be great in settings to have an option to choose which page you want to be the default login page. That gives the most amount of customization with very little work. 

There are plans to also implement a proper 'homescreen' as well, which will show a few headers with things like 'Conntinue Listening', 'Suggested NewPodcasts', and quick links to your most listened Podcasts. 

The first option is of course going to be much faster to implement, but in due time a proper homescreen is certainly planned too.

**Comment by madeofpendletonwool** [comment]
Quick update to this. The groundwork for filtering has been layed. In the podcasts page there's now a filter button and this allows you currently to filter podcasts by category. Next step is going to be to add filtering to all the episode pages, and and figure out everything it would be nice to be able to filter by. Things like partially listened episodes, unlistened episodes, episodes under an hour/specific amount of time,  episodes with specific keywords, etc. Pretty much anything could be filtered. With this groundwork layed on the podcasts page to be able to filter by category at this point we can take this just about anywhere since the filtering logic is now in place.

FYI, there are still plans to set a default homepage. That is still on it's way too


![image](https://github.com/user-attachments/assets/443960ab-0688-426b-b8f5-7b0e55d0d77c)

**Comment by hollisticated-horse** [comment]
I believe a good insperation would be [Antennapod's](https://antennapod.org/) Home page. 

There is (in order of appearance):

- Continue listening (Queue ?)
- See What's New (latest)
- Get suprised (some random thingy ? )
- Check your classics (subscriptions that have been most played)
- Manage Downloads

**Comment by madeofpendletonwool** [comment]
@hollisticated-horse I agree! These are all great ideas and considering all the APIs are already created it's just a matter of creating the frontend layout to do all this.

**Comment by hollisticated-horse** [comment]
I'll have a look at making a mockup :grin: 
I'll post it here.

**Comment by madeofpendletonwool** [comment]
Moved Sorting to it's own issue: #343

**Comment by madeofpendletonwool** [comment]
Both a custom homepage and the ability to change which page you'd like to start on have been added. The custom homepage allows for quick access to various different areas in the app - Along with thing like top podcasts, access to smart playlists, quick recent episodes etc.
